xds/internal/balancer/clusterresolver/e2e_test/balancer_test.go

```diff
@@ -445,3 +446,64 @@ func (s) TestOutlierDetectionConfigPropagationToChildPolicy(t *testing.T) {
 		t.Fatalf("Timeout when waiting for child policy to receive its configuration")
 	}
 }
+
+// Test verifies that LB waits for child policy configuration update
```

Comment:
// TestChildPolicyConfigUpdateBlocking ensures that the load balancer 
// blocks and waits for the child policy configuration to be updated
// in response to configuration changes.

Reply:
@aranjans need to change the comment too along with test name